app/views/questionings/_printable.html.erb

```diff
@@ -1,7 +1,8 @@
 <tr class="question <%= "depth-#{qing.depth}" %><%= qing.condition ? ' conditional' : '' %>">
   <td class="name">
     <% if qing.condition %>
```

Comment:
Can we change this to use the new condition API? There is a method `display_conditionally?` that is perfect for here.